app/config/assets/css/odk-survey.css

```diff
@@ -287,9 +288,58 @@ input[type="range"]::-webkit-slider-thumb:before {
     margin-top: 45px;
 }
 #confirmationPopup .modal-content, #screenPopup .modal-content {
-    background-color: black;
-    color: white;
+    background-color: white;
+}
+
+.modal{
+   display: block;
+   background-color: rgba(0, 0, 0, 0.5);
+ }
+
+.modal-content{
+   border-radius: 10px;
+   border: none;
+   box-shadow: 0 2px 10px rgba(0, 0, 0, 0.1);
+}
+
+.modal-body{
+    padding: 20px;
+}
+
+.modal-body h3 {
+    margin-bottom: 18px!important;
+    font-size: 18px;
+    font-weight: bold;
+    color: black;
```

Comment:
color `black` doesn't align with the design guidelines

#1D1D1D is what should be used here instead